src/txn/txn_rollback_to_stable.c

```diff
@@ -869,10 +872,11 @@ __wt_rollback_to_stable(WT_SESSION_IMPL *session, const char *cfg[])
     F_CLR(session, WT_SESSION_ROLLBACK_TO_STABLE);
 
     /*
-     * Forcibly log a checkpoint after rollback to stable to ensure that both in-memory and on-disk
-     * versions are same.
+     * Forcibly log a checkpoint when not in in-memory configuration after rollback to stable to
```

Comment:
Suggested rewording:
"If the configuration is not in-memory, forcibly log a checkpoint after rollback to stable to ensure that both in-memory and on-disk versions are the same."

Reply:
Fixed.